src/test/java/pages/ApplePage.java

```diff
@@ -0,0 +1,59 @@
+package pages;
+
+import elements.Tag;
+import elements.Text;
+import org.openqa.selenium.TimeoutException;
+import org.openqa.selenium.support.FindBy;
+import org.testng.Assert;
+import properties.Property;
+import util.WaitUtils;
+
+
+public class ApplePage extends BasePage {
+
+    public static final String APPLE_FILTER_ENDPOINT = "mobile?mfr%5B0%5D=apple";
+
+    @FindBy(className = "schema-tags__text")
+    protected Tag tagSign;
+
+    @FindBy(className = "schema-header__title")
+    protected Text headerLabel;
+
+    public ApplePage() {
+        super();
+    }
+
+    @Override
+    public ApplePage openPage() {
+        driver.navigate().to(props.getKeyProperty(Property.valueOf(Property.CATALOGUE_URL + APPLE_FILTER_ENDPOINT)));
```

Comment:
please simplify Property.valueOf(Property.CATALOGUE_URL + APPLE_FILTER_ENDPOINT), why do you need valueOf here? please think of other approaches

Reply:
simplified